djautotask/tests/fixtures.py

```diff
@@ -0,0 +1,41 @@
+
+API_SERVICE_TICKET = {
+    'AccountID': 29683561,
+    'AllocationCodeID': 29683407,
+    'AssignedResourceID': 29683794,
+    'AssignedResourceRoleID': 29683396,
+    'ChangeInfoField1': '',
+    'ChangeInfoField2': '',
+    'ChangeInfoField3': '',
+    'ChangeInfoField4': '',
```

Comment:
Is it common for several of these fields to come in blank in real requests? If so just leave as is, if not, remove fields we wouldn't normally receive as empty strings.

Reply:
Based on the ticket from our sandbox yes, I've only seen them empty. The docs are not clear on what these fields are actually used for. We are not using them now so I can just remove them.

Reply:
No no, do they show up in the replies as empty strings, or do they not show up at all? If they show up as empty strings from the data we receive leave them in the mock data. We want to keep as close to reality as possible.

Reply:
Yes they are returned as empty stings. I'll leave them in the fixture.

Reply:
As for the unclosed file warning. I'm not seeing the warning myself. I think its happening in fixture_utils.py: line 35 when the client object is instantiated. I found an open issue for it at https://bitbucket.org/jurko/suds/issues/102/resourcewarning-unclosed-file-with-python

I'm not sure how to resolve this since the suds library is actually opening the file not us. Otherwise I could just add a proper `with ... open():` etc. Maybe there's something else I'm missing here.

Reply:
Yeah I'm consistently getting this, even on my home workstation. I'll show you tomorrow morning. If you can fix it that simply great. I think this is the only thing left before merging.

Reply:
Because the suds Client is opening the file it limited my options for fixing this unless we open a PR for suds-jurko. I dug around in the suds library and found a possible location for the file not being closed but not sure because we only get the warning once not every time we open the file. So not really sure if I know exactly the location that this. 

Instead of fixing the issue directly, I've moved the client instantiation code to its own function and set as a constant so that the file only gets opened once. This removes the warning but I'm guessing the issue is likely still there. At the very least its probably better to open the file once instead of repeatedly.

Reply:
I think that should be fine, I am not seeing the warning either, and everything is still working.